File: programmers/js_src/level01/12954.js
// x만큼 간격이 있는 n개의 숫자

function solution(x, n) {
  var answer = [];
  var num = x;

  for (let i = 0; i < n; i++) {
    answer.push(num);
    num += x;
  }

  return answer;
}

// 다른 풀이
// function solution(x, n) {
//   return Array(n).fill(x).map((v, i) => (i + 1) * v)
// }

File: programmers/java/level01/12954.java
// x만큼 간격이 있는 n개의 숫자

import java.util.stream.LongStream;

class Solution {
    public long[] solution(int x, int n) {
        return LongStream.range(1, n + 1)
            .map(num -> num * x)
            .toArray();
    }
}